docs_guidelines/how_to/crossreference.rst

```diff
@@ -0,0 +1,82 @@
+================================================
+Crossreference to other pages from documentation
```

Comment:
Can you add a note for avoiding using backquotes and links at the same time in jupyter notebook? This often happens when a developer want to reference a function/class in jupyter notebook because it's a common practice to put those things in backquotes and in local jupyter notebook it looks absolutely fine. But this causes problem due to `nbsphinx` first convert jupyter notebook into rst and then build into html. This is a know limitation of `nbsphinx`. One should See here more details: https://github.com/Qiskit/qiskit-tutorials/pull/1353#discussion_r1042296550

Reply:
Not sure if it really fits here. This how-to is supposed to be for `rst` not notebooks.

Reply:
But this is talking about crossreference in general, not only about crossreference for rst right?

Reply:
You can create a separate how-to guide for cross referencing for jupyter notebooks but I don't think it's necessary to split that separately. I know this is not perfect but I think it should be mentioned here because right now a lot of tutorials are written in jupyter notebook